resources/views/movies/show.blade.php

```diff
@@ -0,0 +1,132 @@
+@extends('layouts/app')
+
+@section('content')
+    <div class="container px-6 py-3 mx-auto">
+        <section class="text-gray-600 body-font">
+            {{--     Movie Summary Section       --}}
+            <div class="container mx-auto flex px-5 py-32 md:flex-row flex-col items-center">
+                <div class="w-full lg:pr-24 md:pr-16 flex flex-col md:items-start md:text-left mb-16 md:mb-0 items-center text-center">
+                    <div class="flex justify-center mb-12">
+                        <div class="flex items-center px-4 py-2 font-medium tracking-wide capitalize transition-colors duration-200 transform rounded-md border-2 border-gray-700 hover:border-gray-500">
+                            <span class="mx-2 whitespace-nowrap">{{ $movie->year }}</span>
+                        </div>
+                        <div class="flex items-center ml-5 px-4 py-2 font-medium text-white tracking-wide capitalize transition-colors duration-200 transform bg-blue-600 rounded-md hover:bg-blue-500 focus:outline-none focus:bg-blue-500">
+                            <svg class="w-5 h-5 mx-1" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="#fff">
+                                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M11.049 2.927c.3-.921 1.603-.921 1.902 0l1.519 4.674a1 1 0 00.95.69h4.915c.969 0 1.371 1.24.588 1.81l-3.976 2.888a1 1 0 00-.363 1.118l1.518 4.674c.3.922-.755 1.688-1.538 1.118l-3.976-2.888a1 1 0 00-1.176 0l-3.976 2.888c-.783.57-1.838-.197-1.538-1.118l1.518-4.674a1 1 0 00-.363-1.118l-3.976-2.888c-.784-.57-.38-1.81.588-1.81h4.914a1 1 0 00.951-.69l1.519-4.674z" />
+                            </svg>
+                            <span class="whitespace-nowrap font-normal"> 4.8 / 10 </span>
+                        </div>
+                    </div>
+                    <h1 class="title-font sm:text-6xl text-5xl mb-4 font-medium text-gray-900"> {{ $movie->title }}  </h1>
+                    <div class="flex justify-center mb-12">
+                        <div class="flex items-center py-2 font-medium tracking-wide">
+                            <span class="mx-2 whitespace-nowrap">
+                                {{ $duration[0] }} hrs
+                                {{ $duration[1] }} min
+                            </span>
+                            <span class="whitespace-nowrap font-normal">
+                                <span class="mx-4 whitespace-nowrap">|</span>
+                                @foreach($genres as $genre)
```

Comment:
Once you consolidate the query, this would then be `$movies->genres as genre`, so I do not this you would need two loops.